recipe/meta.yaml

```diff
@@ -36,20 +36,24 @@ test:
   imports:
     - yarl
   source_files:
+    - pytest.ini
     - tests
   requires:
     - pip
     - pytest
+    - pytest-cov
```

Comment:
```suggestion

```
This seems to be needed because of the `pytest.ini` and we don't typically try to implement test coverage.